### PR TITLE
WE-3096 AnswerGroup component

### PR DESCRIPTION
**Pull request by jswaldon**
Created AnswerGroup component

### CODE REVIEW

File: packages/es-components/src/components/controls/answer-group/AnswerButton.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none; /* Non-prefixed version, currently supported by Chrome and Opera */
+`;
+
+const AnswerInput = styled.input`
+  clip-path: inset(100%);
+  clip: rect(1px 1px 1px 1px); /* IE 6/7 */
```
Comment (Darrken) [inline packages/es-components/src/components/controls/answer-group/AnswerButton.js]: We don't need to support IE 6/7, so this should be safe to remove.

File: packages/es-components/src/components/controls/answer-group/AnswerButton.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  font-size: 16px;
+  font-weight: bold;
+  display: block;
+  padding: 10px 10px 10px 10px;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none; /* Non-prefixed version, currently supported by Chrome and Opera */
```
Comment (Darrken) [inline packages/es-components/src/components/controls/answer-group/AnswerButton.js]: We're probably okay without the comments on these rules.

File: packages/es-components/src/components/controls/answer-group/AnswerButton.js (hunk 1 above)
Comment (jswaldon) [reply packages/es-components/src/components/controls/answer-group/AnswerButton.js]: removed

File: packages/es-components/src/components/controls/answer-group/AnswerButton.js (hunk 2 above)
Comment (jswaldon) [reply packages/es-components/src/components/controls/answer-group/AnswerButton.js]: Removed

**Comment by Darrken** [comment]
There will likely be some style changes coming for this component, but functionally I think it looks good. I'd like to hear @aabenoja and @jrios opinions on how we tackle adding this to the `next` branch since we've got the alpha out now and this will require some rewrite when adding to that version. 

I think this component fits here in `es-components` fine, but Jake might have a different opinion on that.

**Comment by jrios** [comment]
I would definitely go against `next` instead of `master` for this. I would also recommend you start with the alpha release on any brand new downstream projects.

**Pull request closed without merging**